### PR TITLE
Add zanocloud interview

### PR DESCRIPTION
**Pull request by mooreds**

### CODE REVIEW

File: site/_posts/2022-06-03-zanocloud-helps-companies-with-fusionauth.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+*This interview has been lightly edited for clarity and length.*
+
+-------
+
+**Dan:** Can you tell me a bit about ZanoCloud? What is the company's mission?
+
+**Dmitry:** We’re a technology consulting company that brings cloud infrastructure and big data processing back down to earth for people running businesses.
+
+It’s our mission to make it more accessible to everyone. Our team of experts has a wealth of experience, and we provide the best solution for our clients, no matter what their specific needs may be. So, they can focus on running their businesses, not managing their technology.
```
Comment (robotdan) [inline site/_posts/2022-06-03-zanocloud-helps-companies-with-fusionauth.md:24]: > Our team of experts have a wealth of experience,

File: site/_posts/2022-06-03-zanocloud-helps-companies-with-fusionauth.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+**Dan:** How much time and money would you say FusionAuth has saved you?
+
+**Dmitry:** It’s hard to estimate, it was different for every project. But I would say, hundreds/thousands of dollars of service cost for the initial project stages compared to managed services.
+
+Time-wise, it saved several weeks of work and provided a better user experience for user base migration to FusionAuth.
+
+> We choose FusionAuth because of its installation and configuration simplicity.
+
+**Dan:** How do you run FusionAuth (kubernetes, standalone tomcat server, behind a proxy, etc)?
```
Comment (robotdan) [inline site/_posts/2022-06-03-zanocloud-helps-companies-with-fusionauth.md:82]: Suggest replace `standalone tomcat server` with `standalone web app`. We are removing Tomcat in a future release, and the type of app server we use is not pertinent.

File: site/_posts/2022-06-03-zanocloud-helps-companies-with-fusionauth.md (hunk 1 above)
Comment (troutgirlfusionauth) [reply site/_posts/2022-06-03-zanocloud-helps-companies-with-fusionauth.md:24]: Team has, not experts have :)

File: site/_posts/2022-06-03-zanocloud-helps-companies-with-fusionauth.md (hunk 2 above)
Comment (troutgirlfusionauth) [reply site/_posts/2022-06-03-zanocloud-helps-companies-with-fusionauth.md:82]: OK

**Review by robotdan — APPROVED** [review]

**Pull request merged**